### DISCUSSION TITLE
Teach Javascript

### DISCUSSION THREAD

**Discussion by jivthesh** [EddieHubCommunity/support#1398] [Ideas]
It would be very helpful for beginners

**Comment by kaiwalyakoparkar** [comment]
https://codedamn.com/ -> Learn all the topics related to web development for almost free

**Comment by tashasyarifah** [comment]
https://www.freecodecamp.org/ : Highly recommend this. I start my coding journey from there. 

https://javascript30.com/ : Learning how to create things only with vanilla JavaScript in 30 days (I've not try this one yet but I see a lot of people highly recommend this).

https://grasshopper.app/ : Learning Js while you are making a game. It also has great explanation so it doesn't matter if you just started (I've tried this one. It's awesome cause they make it like JavaScript is not scary as what people say).

**Reply by tashasyarifah** [reply]
another go to website is https://javascript.info/ It's more a like documentation but easy to read.

**Comment by kaiwalyakoparkar** [comment]
https://eloquentjavascript.net/ <--- It's a good site too

**Comment by shikharcodes** [comment]
https://www.w3schools.com/js/  :  Will help in learning JavaScript and also will be useful for quick reference
